HACKING.rst

```diff
@@ -0,0 +1,112 @@
+KNoT Network of Things - Gateway - HACKING
+==========================================
+
+Use a Vagrant box (experimental)
```

Comment:
Is this vagrant box functional? I think it's (very) outdated. Perhaps add a note about it to be removed latter after testing.

Reply:
The vagrant box is functional, a member of the team has used already. But @joaoaneto will test the steps to make sure that is correct, a new issue will be created.